include/wx/textctrl.h

```diff
@@ -608,6 +608,69 @@ class WXDLLIMPEXP_CORE wxTextAttr
     wxString            m_listStyleName;
 };
 
+// ----------------------------------------------------------------------------
+// Search features for wxTextCtrl
+// ----------------------------------------------------------------------------
+
+enum class wxTextSearchDirection
+{
+    Down,
+    Up
+};
```

Comment:
Maybe move it inside `wxTextSearch`? This would mean writing `wxTextSearch::Direction::Down` instead of `wxTextSearchDirection::Down`, i.e. 2 extra characters, but it seems more logical to have everything search-related inside this single struct.

Reply:
Done.